### ISSUE TITLE
Is there a mailing list? 

### ISSUE THREAD

**Issue by dims** [dapr/community#536]
Seems like most of the activity is on discord or calls ...

https://github.com/dapr/community/blob/master/README.md#communication-and-discord

has the community tried using mailing list for at least say announcements? did that not work?

Asking because both discord and calls are ephemeral compared to mailing lists. Typically cncf projects have a dev/user/announcement lists, easy to spin them up on https://lists.cncf.io/

**Comment by dims** [comment]
xref: https://github.com/cncf/toc/issues/1354

**Comment by msfussell** [comment]
We did have a mailing list when we first started the project, we just found that we were having to manage questions in two places and this became harder and less clear on where to ask questions. Hence we decide to consolidate to one place and choose Discord since this was more engaging. We could have one for announcements, but would be hesitant to bring this back for questions since we find the current Discord one works well with the community.
